cmake/target.cmake

```diff
@@ -64,6 +64,15 @@ if (CMAKE_CROSSCOMPILING)
 
         set (HAS_POST_2038_EXITCODE "0" CACHE STRING "Result from TRY_RUN" FORCE)
         set (HAS_POST_2038_EXITCODE__TRYRUN_OUTPUT "" CACHE STRING "Output from TRY_RUN" FORCE)
+
+        # FIXME: broken dependencies
+        set (USE_SNAPPY OFF)
+        set (ENABLE_SSL OFF)
```

Comment:
It means that `clickhouse-client` on Mac OS cannot connect to TLS-enabled servers like in Yandex Cloud.